src/main/java/com/exasol/projectkeeper/AbstractProjectFilesWalker.java

```diff
@@ -0,0 +1,95 @@
+package com.exasol.projectkeeper;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import io.github.classgraph.ClassGraph;
+import io.github.classgraph.Resource;
+import io.github.classgraph.ScanResult;
+
+/**
+ * Abstract basis classes that validate the project files.
+ * <p>
+ * This class walks the project templates, and calls a {@link ProjectFileTemplateVisitor} for each template file.
+ * </p>
+ */
+public class AbstractProjectFilesWalker {
```

Comment:
The name says Abstract, but the class is not abstract. Please, Remove Abstract from the name

Reply:
oh, it should be abstract